service/src/LevelZeroIOGroup.cpp

```diff
@@ -294,6 +294,118 @@ namespace geopm
                                   },
                                   1e6
                                   }},
+                              {M_NAME_PREFIX + "GPU_RAS_RESET_COUNT", {
```

Comment:
Please move all of the new signals to the bottom of the m_signal_available initializer list.  New signals go in the bottom, not in the middle.

Reply:
Noted. Fixed.